src/Internal/Types/TxConstraints.purs

```diff
@@ -696,75 +666,3 @@ mustSatisfyAnyOf =
 -- | chain and collateral will be lost.
 mustNotBeValid :: TxConstraints
 mustNotBeValid = singleton $ MustNotBeValid
-
--- | Are the constraints satisfiable given the time intervals?
-isSatisfiable :: TxConstraints -> Boolean
-isSatisfiable (TxConstraints { constraints }) =
-  let
-    intervals =
-      Array.mapMaybe
-        ( case _ of
-            MustValidateIn i -> Just i
-            _ -> Nothing
-        )
-        constraints
-    itvl = foldl intersection always intervals
-  in
-    not (isEmpty itvl)
-
-pubKeyPayments
-  :: TxConstraints
-  -> Array (PaymentPubKeyHash /\ Value)
-pubKeyPayments (TxConstraints { constraints }) =
-  toUnfoldable
-    $ fromFoldableWith (<>)
-    $ constraints >>=
-        case _ of
-          MustPayToPubKeyAddress pkh _ _ _ vl -> Array.singleton (pkh /\ vl)
-          _ -> []
-
--- | The minimum `Value` that satisfies all `MustSpendAtLeast` constraints
-mustSpendAtLeastTotal
-  :: TxConstraints -> Value
-mustSpendAtLeastTotal =
-  foldr (join <<< f) mempty <<< _.constraints <<< unwrap
-  where
-  f :: TxConstraint -> Value
-  f (MustSpendAtLeast v) = v
-  f _ = mempty
-
--- | The minimum `Value` that satisfies all `MustProduceAtLeast` constraints
-mustProduceAtLeastTotal
```

Comment:
`mustProduceAtLeastTotal` and `mustSpendAtLeastTotal` are still mentioned in doc/test-plan.md